test/mocha.opts

```diff
@@ -1,4 +1,5 @@
 --require babel-register
+--require jsdom-global/register
```

Comment:
BTW - I think this should be left to userland to shim, it seems to be a common issue, and no need for us to shim for the user, just the same as we don't babel-polyfill for them either.